packages/pwa-kit-react-sdk/src/config/schemas/v1-0-0.json

```diff
@@ -0,0 +1,264 @@
+                            "$id": "#root/server/mobify/pageNotFoundURL",
+                            "title": "Pagenotfoundurl",
+                            "type": "string",
+                            "default": "",
+                            "examples": ["/page-not-found"],
+                            "pattern": "^.*$"
+                        },
+                        "ssrEnabled": {
+                            "$id": "#root/server/mobify/ssrEnabled",
+                            "title": "Ssrenabled",
+                            "type": "boolean",
+                            "examples": [true],
+                            "default": true
+                        },
+                        "ssrOnly": {
+                            "$id": "#root/server/mobify/ssrOnly",
+                            "title": "Ssronly",
+                            "type": "array",
+                            "default": [],
+                            "items": {
+                                "$id": "#root/server/mobify/ssrOnly/items",
+                                "title": "Items",
+                                "type": "string",
+                                "default": "",
+                                "examples": ["ssr.js"],
+                                "pattern": "^.*$"
+                            }
+                        },
+                        "ssrShared": {
+                            "$id": "#root/server/mobify/ssrShared",
+                            "title": "Ssrshared",
+                            "type": "array",
+                            "default": [],
+                            "items": {
+                                "$id": "#root/server/mobify/ssrShared/items",
+                                "title": "Items",
+                                "type": "string",
+                                "default": "",
+                                "examples": ["static/ico/favicon.ico"],
+                                "pattern": "^.*$"
+                            }
+                        },
+                        "ssrParameters": {
+                            "$id": "#root/server/mobify/ssrParameters",
+                            "title": "Ssrparameters",
+                            "type": "object",
+                            "required": ["ssrFunctionNodeVersion", "proxyConfigs"],
+                            "properties": {
+                                "ssrFunctionNodeVersion": {
+                                    "$id": "#root/server/mobify/ssrParameters/ssrFunctionNodeVersion",
+                                    "title": "Ssrfunctionnodeversion",
+                                    "type": "string",
+                                    "default": "",
+                                    "examples": ["14.x"],
+                                    "pattern": "^.*$"
+                                },
+                                "proxyConfigs": {
```

Comment:
Is it possible to rename this to `localProxyConfigs` and only have it used in the local dev environment?